components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -85,6 +91,13 @@ object Glean {
         ProcessLifecycleOwner.get().lifecycle.addObserver(gleanLifecycleObserver)
     }
 
+    /**
+     * Returns true if the Glean library has been initialized.
+     */
+    fun isInitialized(): Boolean {
```

Comment:
I'm not sure if this is possible at all, but can this be marked as `internal`? I'm wondering if we even need this function, but we certainly don't want it to be accessible as part of the outer facing API.

Reply:
I think it's helpful to have it as a function in case we need to override its semantics later, but I agree it probably makes sense to make it internal.

Reply:
Sounds good to me!